src/vs/workbench/parts/preferences/browser/preferencesService.ts

```diff
@@ -117,32 +117,17 @@ export class PreferencesService extends Disposable implements IPreferencesServic
 	}
 
 	resolveModel(uri: URI): TPromise<IModel> {
-		const workspaceSettingsUri = this.getEditableSettingsURI(ConfigurationTarget.WORKSPACE);
```

Comment:
@sandy081 Given what you said earlier, it should be safe to delete this and `resolveSettingsContentFromWorkspaceConfiguration`, if this method only handles DefaultSettings/KeyboardEditorModels?